frontend/src/features/admin-form/create/workflow/FormWorkflowService.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import {
+  FormSettings,
+  FormWorkflow,
+  FormWorkflowStep,
+  MultirespondentFormSettings,
+} from '~shared/types/form'
+
+import { ApiService } from '~services/ApiService'
+
+import { ADMIN_FORM_ENDPOINT } from '~features/admin-form/common/AdminViewFormService'
+
+export const updateFormWorkflow = async (
+  formId: string,
+  newWorkflowSettings: MultirespondentFormSettings['workflow'],
+) => {
+  return ApiService.patch<FormSettings>(
+    `${ADMIN_FORM_ENDPOINT}/${formId}/settings`,
```

Comment:
-- saw that you have already noted this on the PR, this will be resolved after we ship? Totally fine by it as we have a good amount of buffer post-release to make refinements like this.

Reply:
Yup, in the spirit of making PRs as small as possible to avoid scope creep,, I decided not to do that modification here. This code was already written and I didn't want to touch it too much. Nonetheless, a linear ticket to resolve this has already been opened as [FRM-1632](https://linear.app/ogp/issue/FRM-1632/proper-object-level-rest-api-for-workflow-steps)!

Reply:
Great division of scope btw! Features are cleanly cut, and isolated.